### ISSUE TITLE
Object keys alphabet order

### ISSUE THREAD

**Issue by Diokuz** [eslint/eslint#4947]
I want an option for such case:

```js
// Valid
var qwe = {
    a: 1,
    b: 2,
    c: 3
}

// Invalid
var qwe = {
    b: 2,
    a: 1,
    c: 3
}
```

Is that possible right now?

**Comment by eslintbot** [comment]
@Diokuz Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by nzakas** [comment]
We do not have a rule for this. If you'd like, you can create a custom rule to do this.
